### DISCUSSION TITLE
2022-04-20 Weekly Meeting 52 Agenda

### DISCUSSION THREAD

**Discussion by antpb** [omigroup/omigroup#216] [Weekly Meetings]
Weekly Meeting 52 is happening on Wednesday April 20th, 2022 from 9:00 AM PST(16:00 UTC) - 10:00 AM PST (17:00 UTC)

You can add it to your calendar [here](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).
 
We'll be meeting again in the omi-weekly-meeting channel of the [AngellXR/OMI Discord](https://discord.gg/NJtT9grz5E).

## Who we are and why we're here:

The Open Metaverse Interoperability Group (OMI) is focused on bridging virtual worlds by designing and promoting protocols for identity, social graphs, inventory, and more. Our members include businesses and individuals working towards this common goal. 

If you’re an observer or here for the first time, this is a meeting where people who work on the OMI group initiatives discuss updates and contribution opportunities for current projects and goals. This is your chance to jump in and help!

We welcome you to introduce yourself in #omi-meeting-chat as we dive into the agenda items. There will be an open floor in the last 15 minutes of the meeting (agenda permitting) where we can discuss any items you feel are relevant to the current or near future OMI goals.

## Agenda

- Volunteer to take notes? Volunteer to record? 
- Welcome newcomers

## Announcements: 
[OMI Elections](https://github.com/omigroup/omigroup/discussions/211)

Working Group Updates (If we have time):

- OMI Website update
- OMI-Chan group
- Scripting group
- glTF Extensions Group
- https://github.com/omigroup/media
- any others?

**Comment by mrmetaverse** [comment]
Blog: Celebrating our one year!  https://github.com/omigroup/website/issues/54 @robertlong @LightLodges @antpb @pcsmgnt

**Comment by mrmetaverse** [comment]
Website group and media group to Collab on blogging

**Comment by mrmetaverse** [comment]
Elections live in discord under #omi-elections channel for 24 hours

**Comment by mrmetaverse** [comment]
https://github.com/omigroup/media/issues/20

Define omigroup/media roadmap

**Reply by pcsmgnt** [reply]
Done, pending alignment review

**Comment by robertlong** [comment]
Note: Create a jobs board utilizing Airtable as an input method or just wordpress posts

**Comment by mrmetaverse** [comment]
Reminder we have a #metaverse-jobs-board in the discord! 

Also talking about a form for adding jobs to be done 🚀
